### PR TITLE
[device/dx010] configuration port index start from 1

### PR DESCRIPTION
**Pull request by pphuchar**
**- What I did**
Update following SKU's port index to start from 1.
 - Seastone-DX010-10-50
 - Seastone-DX010-50
 - Seastone-DX010

**- How I did it**
- Update SKU's port_config.ini
- Update sfputil.py plugin

**- How to verify it**
Run sfputil  ...  to verify all function work properly.

**- Description for the changelog**

Device Dx010 update port index start from 1.

### CODE REVIEW

File: device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py [hunk 1]
```diff
@@ -41,7 +41,8 @@ def __init__(self):
 
         for x in range(self.PORT_START, self.PORT_END + 1):
             if self.port_start == 1:
-                self.port_to_eeprom_mapping[x] = eeprom_path.format((x - 1) + 26)
+                self.port_to_eeprom_mapping[
+                    x] = eeprom_path.format((x - 1) + 26)
```
Comment (yxieca) [inline device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py]: This line break looks like an accident to me. Can you fix it?

File: device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py [hunk 2]
```diff
@@ -83,7 +84,8 @@ def get_low_power_mode(self, port_num):
             return False
 
         try:
-            reg_file = open("/sys/devices/platform/dx010_cpld/qsfp_lpmode", "r")
+            reg_file = open(
+                "/sys/devices/platform/dx010_cpld/qsfp_lpmode", "r")
```
Comment (yxieca) [inline device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py]: Can you remove above link break?

File: device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py [hunk 3]
```diff
@@ -109,7 +111,95 @@ def get_low_power_mode(self, port_num):
...
 
     def set_low_power_mode(self, port_num, lpmode):
-        raise NotImplementedError
+        # Check for invalid port_num
+        if port_num < self.port_start or port_num > self.port_end:
+            return False
+
+        try:
+            reg_file = open(
+                "/sys/devices/platform/dx010_cpld/qsfp_lpmode", "r+")
```
Comment (yxieca) [inline device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py]: Please merge above 2 lines into one.

File: device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py [hunk 4]
```diff
@@ -109,7 +111,95 @@ def get_low_power_mode(self, port_num):
...
 
     def reset(self, port_num):
-        raise NotImplementedError
+        # Check for invalid port_num
+        if port_num < self.port_start or port_num > self.port_end:
+            return False
+
+        try:
+            reg_file = open(
+                "/sys/devices/platform/dx010_cpld/qsfp_reset", "r+")
```
Comment (yxieca) [inline device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py]: Please merge above 2 lines into one.

File: device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py (hunk 1 above)
Comment (pphuchar) [reply device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py]: Fixed in 5db212356dfce548f532d8398de43fed8dccf973

File: device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py (hunk 2 above)
Comment (pphuchar) [reply device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py]: Fixed in 5db212356dfce548f532d8398de43fed8dccf973

File: device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py (hunk 3 above)
Comment (pphuchar) [reply device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py]: Fixed in 5db212356dfce548f532d8398de43fed8dccf973

File: device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py (hunk 4 above)
Comment (pphuchar) [reply device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py]: Fixed in 5db212356dfce548f532d8398de43fed8dccf973

**Comment by pphuchar** [comment]
@zhenggen-xu  This index change is requested by @yxieca.
The front panel port index of dx010 is started from 1.

**Comment by lguohan** [comment]
@zhenggen-xu , nothing is broken. is it refactoring the sfp plugin code and make index align with other platforms, starting from 1. If you do not extra dependency on this index, there should be no concern.

**Comment by yxieca** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Pull request merged**